### ISSUE TITLE
head-support.js comment claims it's an htmx 1.0 extension

### ISSUE THREAD

**Issue by scrhartley** [bigskysoftware/htmx-extensions#46]
The [extensions site](https://extensions.htmx.org/) says it's for htmx 2.0 extensions and links to this repo,
but the comment at the top for the extension's source says it's for version 1.0 of htmx.

See:
https://github.com/bigskysoftware/htmx-extensions/blob/main/src/head-support/head-support.js

Comment at top of file:
```javascript
//==========================================================
// head-support.js
//
// An extension to htmx 1.0 to add head tag merging.
//==========================================================
```

**Comment by Telroshan** [comment]
Hey, good catch, this was likely an oversight, or a residue of the time when we thought head-support would be embedded in htmx2, which eventually didn't happen.

All extensions on this repo are for htmx 2, feel free to open a PR to remove that version number mention!

**Comment by Telroshan** [comment]
Fixed in #47
